cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/StateVariablesExportTest.java

```diff
@@ -237,6 +239,58 @@ void microGridBEWithHiddenTapChangers() throws XMLStreamException {
         assertTrue(sv.contains(hiddenTapChangerId));
     }
 
+    @Test
+    void cgmes3MiniGridwithTransformersWithRtcAndPtc() throws IOException, XMLStreamException {
+
+        Network network = ConversionUtil.networkModel(Cgmes3Catalog.miniGrid(), new Conversion.Config());
+
+        // Add a PTC
+        TwoWindingsTransformer twt = network.getTwoWindingsTransformer("813365c3-5be7-4ef0-a0a7-abd1ae6dc174");
+        twt.newPhaseTapChanger()
+                .setLowTapPosition(0)
+                .setTapPosition(0)
+                .beginStep()
+                .setR(0.0)
+                .setX(0.0)
+                .setB(0)
+                .setG(0)
+                .setRho(1.0)
+                .setAlpha(20)
+                .endStep()
+                .add();
+        // Change the RTC tap position and add a PTC
+        ThreeWindingsTransformer t3wt = network.getThreeWindingsTransformer("411b5401-0a43-404a-acb4-05c3d7d0c95c");
+        t3wt.getLeg1().getRatioTapChanger().setTapPosition(16);
+        t3wt.getLeg1().newPhaseTapChanger()
+                .setLowTapPosition(0)
+                .setTapPosition(1)
+                .beginStep()
+                .setR(0.0)
+                .setX(0.0)
+                .setB(0)
+                .setG(0)
+                .setRho(1.0)
+                .setAlpha(10)
+                .endStep()
+                .beginStep()
+                .setR(0.0)
+                .setX(0.0)
+                .setB(0)
+                .setG(0)
+                .setRho(1.0)
+                .setAlpha(20)
+                .endStep()
+                .add();
+
+        // as all the tap positions are unique we check that RTC and PTC tap positions are in the sv file
```

Comment:
It is just a unit test but the assertion are not really clean.

Reply:
it was the simplest option to avoid re-importing the case and checking the tap position (costly in testing time), as the change in the code was very simple and only affected the SV output, but we will implement a more clear alternative.